pkg/pillar/cmd/baseosmgr/baseosmgr.go

```diff
@@ -434,11 +437,10 @@ func handleGlobalConfigModify(ctxArg interface{}, key string,
 		log.Infof("handleGlobalConfigModify: ignoring %s\n", key)
 		return
 	}
-	var gcp *types.GlobalConfig
-	debug, gcp = agentlog.HandleGlobalConfig(ctx.subGlobalConfig, agentName,
-		debugOverride)
+	var gcp *types.ConfigItemValueMap
+	debug, gcp = agentlog.HandleGlobalConfig(ctx.subGlobalConfig, agentName, debugOverride)
 	if gcp != nil {
-		ctx.globalConfig = gcp
+		ctx.configItemValueMap = gcp
```

Comment:
Can gcp ever be nil?? If yes - is iit an error? Add an else case if we need the if here.